pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/SierraTransformer.scala

```diff
@@ -0,0 +1,37 @@
+package uk.ac.wellcome.platform.transformer.sierra
+
+import grizzled.slf4j.Logging
+
+import uk.ac.wellcome.models.transformable.sierra.SierraBibNumber
+import uk.ac.wellcome.platform.transformer.sierra.source.SierraBibData
+
+trait Info {
+  def info: String
+}
+
+trait SierraTransformer extends Logging {
+
+  type Output
+
+  type TransformInfo <: Info
+
+  case class Log(bibId: SierraBibNumber, transformerName: String, transformInfo: TransformInfo, output: Output) {
```

Comment:
Should this be a `SierraTransformerLog` as we will probably be logging from elsewhere?
Although maybe best to look at the structure of that when we're doing it.

Reply:
Well the `Log` is defined within the `SierraTransformer` class so to access it from outside you would have to do `SierraTransformer.Log`, which is maybe nicer than `SierraTransformer.SierraTransformerLog`?

Might make sense to define a more general `Log` trait like you suggested, tho until we do `toJson` or something it will just have `toString` (which is kind of redundant as every type already defines that)